src/N98/Magento/Command/Developer/EmailTemplate/UsageCommand.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace N98\Magento\Command\Developer\EmailTemplate;
+
+use N98\Magento\Command\AbstractMagentoCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use N98\Util\Console\Helper\Table\Renderer\RendererFactory;
+
+class UsageCommand extends AbstractMagentoCommand
+{
+    /**
+     * @var array
+     */
+    protected $infos;
+
+    protected function configure()
+    {
+        $this
+            ->setName('dev:email-template:usage')
+            ->setDescription('Display database transactional email template usage')
+            ->addOption(
+                'format',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Output Format. One of [' . implode(',', RendererFactory::getFormats()) . ']'
+            )
+        ;
+    }
+
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return int|void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->detectMagento($output, true);
+        $this->initMagento();
+        $this->findEmailTemplates();
+
+        if (!empty($this->infos)) {
+            $this->getHelper('table')
+                ->setHeaders(array('id', 'Name', 'Scope', 'Scope Id', 'Path'))
+                ->renderByFormat($output, $this->infos, $input->getOption('format'));
+        } else {
+            $output->writeln("No transactional email templates stored in the database.");
+        }
+    }
+
+    protected function findEmailTemplates()
+    {
+        $templates = \Mage::getModel('adminhtml/email_template')->getCollection();
+
+        foreach ($templates as $template){
+
+            /**
+             * Some modules overload the template class so that the method getSystemConfigPathsWhereUsedCurrently
+             * is not available, this is a workaround for that
+             */
+            if (!method_exists($template, 'getSystemConfigPathsWhereUsedCurrently')){
```

Comment:
Nice internal. Please import the magento core class (via use statment) so that it does not need a trailing \ here.